Documentation/contributing/testing/e2e.rst

```diff
@@ -460,7 +460,7 @@ An example invocation is
 
 ::
 
-  CNI_INTEGRATION=eks K8S_VERSION=1.13 ginkgo --focus="K8s*" -noColor -- -cilium.provision=false -cilium.kubeconfig=`echo ~/.kube/config` -cilium.image="quay.io/cilium/cilium:latest" -cilium.operator-image="quay.io/cilium/operator:latest" -cilium.passCLIEnvironment=true
+  CNI_INTEGRATION=eks K8S_VERSION=1.13 ginkgo --focus="K8s*" -noColor -- -cilium.provision=false -cilium.kubeconfig=`echo ~/.kube/config` -cilium.image="docker.io/cilium/cilium:latest" -cilium.operator-image="docker.io/cilium/operator:latest" -cilium.passCLIEnvironment=true
```

Comment:
just `cilium/cilium:latest` is also enough without the prefix

Reply:
yeah but i wasn't sure if https://github.com/cilium/cilium/blob/da9a72463ee7fd88da6107864eb8dbb42613f243/test/helpers/kubectl.go#L219 can handle image names without domain name.